### PR TITLE
Add cinco de trino blog

### PR DESCRIPTION
**Pull request by bitsondatadev**

### CODE REVIEW

File: _posts/2022-05-17-cinco-de-trino-recap.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
+---
+layout: post
+title:  "Cinco de Trino Recap: Learn how to build a efficient data lake"
+author: Brian Olsen, Brian Zhan
+excerpt_separator: <!--more-->
+---
+
+When Trino (formerly PrestoSQL) arrived on the scene almost 10 years ago, it
+immediately became known for quick analysis of big data in a data warehouse. The
```
Comment (arhimondr) [inline _posts/2022-05-17-cinco-de-trino-recap.md:9]: > of big data in a data warehouse

Has Trino ever been positioned as a data warehouse solution? Maybe rephrase?

File: _posts/2022-05-17-cinco-de-trino-recap.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+miss.
+
+<!--more-->
+
+### Starburst Galaxy lab
+
+Starburst Galaxy enables you to just run a data lake architecture rather than 
+spend your time doing analytics. Skip the infrastructure setup, maintenance, 
+security controls,and  IAM roles. Trino co-creator, Dain Sundstrom walks you
+through the journey of how to use Starburst Galaxy, the Trino SaaS, to analyze
```
[the remark this answers is not in the document]
Comment (bitsondatadev) [reply _posts/2022-05-17-cinco-de-trino-recap.md:34]: https://db-engines.com/en/ranking

File: _posts/2022-05-17-cinco-de-trino-recap.md (hunk 1 above)
Comment (bitsondatadev) [reply _posts/2022-05-17-cinco-de-trino-recap.md:9]: Trino was a replacement for Apache Hive, which is positioned as a data warehouse solution. I could spell that out so that people know what I'm talking about.

https://hive.apache.org/

File: _posts/2022-05-17-cinco-de-trino-recap.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+[Cinco de Trino playlist](https://www.youtube.com/playlist?list=PLFnr63che7wYDHjUsmp43THLmAlqPDHlM). 
+
+In this post, I’d like to cover the key items from each talk you won’t want to 
+miss.
+
+<!--more-->
+
+### Starburst Galaxy lab
+
+Starburst Galaxy enables you to just run a data lake architecture rather than
```
[the remark this answers is not in the document]
Comment (bitsondatadev) [reply _posts/2022-05-17-cinco-de-trino-recap.md:31]: lol I didn't notice this...let me update

File: _posts/2022-05-17-cinco-de-trino-recap.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (arhimondr) [reply _posts/2022-05-17-cinco-de-trino-recap.md:34]: Do you think it is worth providing a link for people like me? :-)

File: _posts/2022-05-17-cinco-de-trino-recap.md (hunk 1 above)
Comment (arhimondr) [reply _posts/2022-05-17-cinco-de-trino-recap.md:9]: It sounds a little strange to me. I wonder if we can drop the `in a data warehouse` and simply leave it as `for quick analysis of big data.`. It somehow reads as "Trino was built for analysis of data stored in a data warehouse (e.g.: Teradata)", but maybe it's just me.

File: _posts/2022-05-17-cinco-de-trino-recap.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (bitsondatadev) [reply _posts/2022-05-17-cinco-de-trino-recap.md:34]: added this link and some explanation

File: _posts/2022-05-17-cinco-de-trino-recap.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+2. Having a really awesome UI to manage expectations.
+3. The data source view that makes it easy to dynamically test your custom
+data quality checks against backends.
+
+<iframe width="720" height="405" src="https://www.youtube.com/embed/9HE6LawCHP8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe> 
+
+### Bring your data into your data lake with Airbyte
+
+The first step of doing any analytics is bringing your data into the data lake.
+FiveTran was a gamechanger for centralizing your data in the data lake. But what
```
[the remark this answers is not in the document]
Comment (bitsondatadev) [reply _posts/2022-05-17-cinco-de-trino-recap.md:84]: removing Fivetran mention

**Review by arhimondr — APPROVED** [review]
